src/olympia/translations/management/commands/process_translations.py

```diff
@@ -1,28 +1,17 @@
-import os
-
 from django.db.models import Q
 
 from olympia.amo.management import ProcessObjectsCommand
 from olympia.translations.models import Translation
-from olympia.translations.tasks import update_outgoing_url
 
 
 class Command(ProcessObjectsCommand):
     def get_model(self):
         return Translation
 
     def get_tasks(self):
-        # Change this on dev/stage using an environement variable. The trailing
-        # slash is important (matches how settings.REDIRECT_URL is set).
-        old_outgoing_url = os.environ.get(
-            'OLD_OUTGOING_URL', 'https://outgoing.prod.mozaws.net/v1/'
-        )
         return {
-            'update_outgoing_url': {
-                'task': update_outgoing_url,
-                'queryset_filters': [
-                    Q(localized_string_clean__icontains=old_outgoing_url),
-                ],
-                'kwargs': {'old_outgoing_url': old_outgoing_url},
+            'copy_spanish_translations': {
```

Comment:
This change is also a bit opaque..

Reply:
Err I should remove this. I removed the old `update_outgoing_url` task because I had planned to add a management command to do the migration but decided against it eventually. I'll revert this bit, it doesn't matter either way.

Reply:
3724bb2ee40dcf49c4d5f55afd39c1f9d35fbf61